packages/blocks/stopwatch/src/app.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { useEffect, useState, useCallback } from "react";
+import { BlockComponent } from "blockprotocol/react";
+
+type AppProps = {
+  start: Date;
+  laps: number[];
+};
+
+function formatDuration(duration: number) {
+  const MILLISECONDS = 1;
+  const SECONDS = 1000 * MILLISECONDS;
+  const MINUTES = 60 * SECONDS;
+  const HOURS = 60 * MINUTES;
+  const DAYS = 24 * HOURS;
+
+  const millis = Math.floor((duration % SECONDS) / 10)
+    .toString()
+    .padStart(2, "0");
+  const seconds = Math.floor((duration % MINUTES) / SECONDS)
+    .toString()
+    .padStart(2, "0");
+  const minutes = Math.floor((duration % HOURS) / MINUTES)
+    .toString()
+    .padStart(2, "0");
+  const hours = Math.floor((duration % DAYS) / HOURS);
+  const days = Math.floor(duration / DAYS);
+
+  if (duration >= DAYS) {
+    return `${days}:${hours}:${minutes}:${seconds}.${millis}`;
+  } else if (duration >= HOURS) {
+    return `${hours}:${minutes}:${seconds}.${millis}`;
+  } else {
+    return `${minutes}:${seconds}.${millis}`;
+  }
+}
+
+function lastLap(laps: number[]): number {
```

Comment:
It makes it quicker to understand what the function is doing if its name starts with a verb. It's also easier to distinguish data `fooBar` from the transormers `doSomethingWithFooBar`.

I guess it’s `getLastLap` in this case.

Reply:
Habit of mine; in Rust, getters usually don't have `get`. Changed to `getLastLap` and converted to arrow-style function in 228a92a3af2914bd6be00d7ee08eb80d05b72aaa